<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QWidget" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Login</string>
  </property>
  <widget class="QLabel" name="lUsuario">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>110</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Usuario</string>
   </property>
  </widget>
  <widget class="QLabel" name="lClave">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>110</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Clave</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leUsuario">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>140</y>
     <width>113</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="leClave">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>140</y>
     <width>113</width>
     <height>28</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::EchoMode::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="lTemperatura">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>260</y>
     <width>381</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbIngresar">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>200</y>
     <width>83</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Ingresar</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
